gax-java/gax/src/test/java/com/google/api/gax/rpc/EndpointContextTest.java

```diff
@@ -339,6 +340,7 @@ void endpointContextBuild_gdchFlow_noUniverseDomain_customEndpoint() throws IOEx
   // For this test running locally or in CI, check that the Env Var is set properly.
   // This test should only run when the maven profile `EnvVarTest` is enabled.
   @Test
+  @SetEnvironmentVariable(key = "GOOGLE_CLOUD_UNIVERSE_DOMAIN", value = "random.com")
```

Comment:
If possible, can the key be something like `key = EndpointContext.GOOGLE_CLOUD_UNIVERSE_DOMAIN` using the constant from https://github.com/googleapis/sdk-platform-java/blob/cf44b0754258b18fe3ae29725e9713b548beff80/gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java#L62

Reply:
Done.